icons/matplotlib/matplotlib-plain.svg

```diff
@@ -0,0 +1 @@
64s-6.449 32.131-18.159 43.841S80.561 126 64 126zM64 3.82c-16.074 0-31.187 6.26-42.553 17.626C10.08 32.813 3.82 47.925 3.82 64s6.26 31.188 17.626 42.554S47.925 124.18 64 124.18s31.188-6.26 42.554-17.626S124.18 80.075 124.18 64s-6.26-31.187-17.626-42.554C95.187 10.08 80.075 3.82 64 3.82z"/><path fill="#11557c" d="M64 112.35c-26.66 0-48.35-21.689-48.35-48.35S37.34 15.65 64 15.65 112.35 37.34 112.35 64 90.66 112.35 64 112.35zm0-96c-26.274 0-47.65 21.375-47.65 47.65S37.726 111.65 64 111.65 111.65 90.274 111.65 64 90.274 16.35 64 16.35z"/><path fill="#11557c" d="M64 98.35c-18.94 0-34.35-15.41-34.35-34.35S45.06 29.65 64 29.65 98.35 45.06 98.35 64 82.94 98.35 64 98.35zm0-68c-18.555 0-33.65 15.095-33.65 33.65S45.445 97.65 64 97.65 97.65 82.555 97.65 64 82.555 30.35 64 30.35z"/><path fill="#11557c" d="M64 84.35c-11.221 0-20.35-9.129-20.35-20.35 0-11.221 9.129-20.35 20.35-20.35 11.221 0 20.35 9.129 20.35 20.35 0 11.221-9.129 20.35-20.35 20.35zm0-40c-10.835 0-19.65 8.815-19.65 19.65S53.165 83.65 64 83.65 83.65 74.835 83.65 64 74.835 44.35 64 44.35z"/><path fill="#11557c" d="M64 71.35c-4.053 0-7.35-3.297-7.35-7.35s3.297-7.35 7.35-7.35 7.35 3.297 7.35 7.35-3.297 7.35-7.35 7.35zm0-14c-3.667 0-6.65 2.983-6.65 6.65s2.983 6.65 6.65 6.65 6.65-2.983 6.65-6.65-2.983-6.65-6.65-6.65z"/><path fill="#11557c" d="m77.551 64.198-13.576-.001a.414.414 0 0 1-.128-.806l12.912-4.194a.414.414 0 0 1 .521.264v.001c.455 1.398.685 2.853.684 4.323a.413.413 0 0 1-.413.413z"/><path fill="#11557c" d="M63.975 64.197a.415.415 0 0 1-.387-.558L77.899 25.51a.413.413 0 0 1 .532-.242 40.996 40.996 0 0 1 11.196 6.351.414.414 0 0 1 .066.581L64.298 64.041a.413.413 0 0 1-.323.156z"/><path fill="#11557c" d="M63.975 64.197a.414.414 0 0 1-.317-.148L28.857 22.364a.412.412 0 0 1 .05-.58l.002-.002A54.784 54.784 0 0 1 51.801 10.44a.414.414 0 0 1 .495.312l12.083 52.941a.415.415 0 0 1-.22.463.433.433 0 0 1-.184.041z"/><path fill="#11557c" d="M17.224 72.681a.414.414 0 0 1-.406-.339c-1.813-9.884-.401-20.309 3.977-29.355a.413.413 0 0 1 .552-.193l42.808 20.615a.413.413 0 0 1-.106.78l-46.75 8.485a.318.318 0 0 1-.075.007z"/><path fill="#11557c" d="M41.657 79.659a.42.42 0 0 1-.341-.178 27.656 27.656 0 0 1-2.177-3.738.414.414 0 0 1 .192-.552h.001l24.462-11.78a.413.413 0 0 1 .415.713L41.893 79.586a.414.414 0 0 1-.236.073z"/><path fill="#11557c" d="M63.99 98.132c-2.58 0-5.165-.287-7.66-.857a.416.416 0 0 1-.311-.496l7.552-33.088c.044-.192.223-.354.415-.321a.412.412 0 0 1 .396.344l5.685 33.46a.414.414 0 0 1-.339.477 34.264 34.264 0 0 1-5.738.481z"/><path fill="#11557c" d="M97.832 106.652a.414.414 0 0 1-.324-.156L63.651 64.041a.413.413 0 0 1 .577-.584l42.833 33.379c.18.141.213.401.072.581a54.696 54.696 0 0 1-9.045 9.146.41.41 0 0 1-.256.089z"/><path fill="#11557c" d="M64.35 125.35h-.7V64.845l-42.784 42.783-.495-.495.248-.247L63.155 64.35H2.65v-.7h60.505L20.372 20.866l.495-.495L63.65 63.155V2.65h.7v60.505l42.784-42.783.495.495-.248.248L64.845 63.65h60.505v.7H64.845l42.783 42.784-.495.495-.247-.248L64.35 64.845z"/></svg>
```

Comment:
Please merge the paths if possible in accordance to the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)